### ISSUE TITLE
6 tests failed. diff:diff key after (2685.8) diff:diff key before (2686.8) diff:diff key before singular (2687.8) diff:diff key before abbreviated (2688.8) diff:diff overflow (2691.2) diff:diff between utc and local (2692.5) 

### ISSUE THREAD

**Issue by CetinBasoz** [moment/moment#3879]
### Client info
```
Date String   : Fri Mar 31 2017 16:35:57 GMT+0300 (+03)
Locale String : 3/31/2017, 4:35:57 PM
Offset        : -180
User Agent    : Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/602.4.8 (KHTML, like Gecko) Version/10.0.3 Safari/602.4.8
Moment Version: 2.18.1
```

====

### diff:diff key after (2685.8) 

hour diff

```javascript
// Expected -4
// Actual   -5
-5 === -4
```

====

### diff:diff key before (2686.8) 

hour diff

```javascript
// Expected 4
// Actual   5
5 === 4
```

====

### diff:diff key before singular (2687.8) 

hour diff singular

```javascript
// Expected 4
// Actual   5
5 === 4
```

====

### diff:diff key before abbreviated (2688.8) 

hour diff abbreviated

```javascript
// Expected 4
// Actual   5
5 === 4
```

====

### diff:diff overflow (2691.2) 

hour diff

```javascript
// Expected 24
// Actual   25
25 === 24
```

====

### diff:diff between utc and local (2692.5) 

hour diff

```javascript
// Expected 4
// Actual   5
5 === 4
```

**Comment by marwahaha** [comment]
@CetinBasoz - do you still get these errors?

**Comment by CetinBasoz** [comment]
Sorry I was sick and couldn't reply earlier. Yes I still get this errors with Safari on MacOS High Sierra. On Mac, I also tried with firefox, opera and chrome (Chrome 63.0.3239.108 64bits). They passed the tests. In the same location (Date String   : Thu Jan 04 2018 19:10:09 GMT+0300 (Turkey Standard Time)) using a windows machine, firefox and edge passes. Chrome shows 2292 failures (report was too long I regretted to submit) - Chrome 63.0.3239.108 64-bits. Our timezone is problematic I know + it looks like it is OS and browser related.
